### PR TITLE
Disables Moldies until a rework is made.

### PR DESCRIPTION
**Pull request by SpaceVampire**
## Why It's Good For The Game
Moldies are unfun, anti-RP, and round ending. It has got to the point where the best way to deal with them is often just wall up the area if they spawn in maint, and decide that it has just been lost. It also loves to spawn in the middle of perma with people around (something blob can't do), which is kind of crap if you had anything going on there. 

Blob is fine, because they don't spawn in pairs. They provide a more engaging fight for both groups, and don't spawn mobs that use broken/buggy mechanics that end up turning half of the station into an unending smoke cloud. While leaving behind a bunch of weird corpses that reply the death animation whenever they came off/on your screen.

Moldies have no benefit, no positive upside, and are all negatives. 
They have the same reason for being in the game as an event that suddenly gibs everyone on station with no warning.

I've heard mention of a possible rework, but until then, they should be disabled to prevent any more round-ruining.

## Changelog
:cl:
del: Moldies.
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
Misleading changelog?

**Comment by SpaceVampire** [comment]
> Misleading changelog?

The two really are just one in the same, hell-spawn of our own creation.
But yeah, I might have blanked out for a very brief moment.
Now sssssh. It is as it never was.

**Comment by Gandalf2k15** [comment]
Selfupvoting unbased.

**Comment by SpaceVampire** [comment]
> Selfupvoting unbased.

It's called priming the pump.

**Comment by Gandalf2k15** [comment]
Rework is not needed for these, weight reduction is.

File: modular_skyrat/modules/biohazard_blob/code/mold_event.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "Moldies"
 	typepath = /datum/round_event/mold
 	weight = 5
-	max_occurrences = 1
+	max_occurrences = 0 // SKYRAT EDIT - disables moldies until a rework is made - Moldies are unfun, anti-RP, and round ending. It has got to the point where the best way to deal with them is often just wall up the area if they spawn in maint, and decide that it has just been lost. It also loves to spawn in the middle of perma with people around (something blob can't do), which is kind of crap if you had anything going on there. Blob is fine, because they don't spawn in pairs. They provide a more engaging fight for both groups, and don't spawn mobs that use broken/buggy mechanics that end up turning half of the station into an unending smoke cloud. While leaving behind a bunch of weird corpses that reply the death animation whenever they came off/on your screen. Moldies have no benefit, no positive upside, and are all negatives. They have the same reason for being in the game as an event that suddenly gibs everyone on station with no warning. I've heard mention of a possible rework, but until then, they should be disabled to prevent any more round-ruining.
```
Comment (FlamingLily) [inline modular_skyrat/modules/biohazard_blob/code/mold_event.dm:5]:
```suggestion
	max_occurrences = 0 // Disabled until rework
```
Yall, this is *modular*, we don't need a skyrat edit

Comment (FlamingLily) [reply modular_skyrat/modules/biohazard_blob/code/mold_event.dm:5]: Especially not **that**

Comment (tf-4) [reply modular_skyrat/modules/biohazard_blob/code/mold_event.dm:5]: Proper documentation is still important

**Comment by Gandalf2k15** [comment]
https://github.com/Skyrat-SS13/Skyrat-tg/pull/7853

**Pull request closed without merging**